adapters/huaweiads/huaweiads.go

```diff
@@ -1169,6 +1173,9 @@ func (a *adapter) extractAdmNative(adType int32, content *content, bidType openr
 				dataObject.Label = "desc"
 				dataObject.Value = getDecodeValue(content.MetaData.Description)
 			}
+			if asset.Data.Type == native1.DataAssetTypeCTAText {
+				dataObject.Value = getDecodeValue(content.MetaData.Cta)
+			}
```

Comment:
Is this change covered by a test case?

Reply:
this insctance was missing, it is covered by test case